CRM/Prospect/Service/SalesOpportunityTrackingMenu.php

```diff
@@ -20,27 +22,39 @@ public function getSubmenus($caseTypeCategoryName, array $permissions = NULL) {
       $permissions = (new CaseCategoryPermission())->get($caseTypeCategoryName);
     }
 
+    $caseTypeCategoryName = ($caseTypeCategoryName === CaseTypeCategory::PROSPECT_CASE_TYPE_CATEGORY_NAME) ?
+      'prospect' :
+       $caseTypeCategoryName;
+
```

Comment:
Do we need this check? Why can't we use the `$caseTypeCategoryName` directly?

Reply:
Yes, because the Prospect the category name is `Prospecting`, but the menu is `Prospect`. Thats why this is necessary. but for newly created categories its fine to use `$caseTypeCategoryName` directly.